studio/components/interfaces/Database/Backups/BackupsList.tsx

```diff
@@ -43,7 +45,11 @@ const BackupsList: FC<Props> = ({}) => {
   return (
     <div className="space-y-6">
       {!sortedBackups?.length && tierKey !== 'FREE' ? (
-        <BackupsEmpty />
+        !isPitrEnabled ? (
+          <BackupsEmpty />
+        ) : (
+          <></>
+        )
       ) : (
         <>
           {!canTriggerScheduledBackups && (
```

Comment:
Open for discussion: Should we still show the old full backups, if there are any? It may make sense to display them, if it's very recent backups. If it's older than a week and PITR is enabled, might not make so much sense anymore

Reply:
yeah, I think that's fair!